client/src/components/ArtistSidebar/FollowButton.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styledComponents from './styledArtist';
+
+const FollowButton = (props) => {
+  const { followStatus } = props;
+  const { changeStatus } = props;
+  const { FolButton } = styledComponents;
+  if (followStatus) {
+    return (
+      <FolButton followStatus={followStatus} onClick={changeStatus}>
+        Following
+      </FolButton>
+    );
+  }
+  return (
+    <FolButton followStatus={followStatus} onClick={changeStatus}>
+      Follow
```

Comment:
Why not define a variable before your return statement, with inner text? I.E,

const innerText = followStatus ? 'Following' : 'Follow'

Then, instead of conditionally rendering the whole FolButton, you can say (I think)

      <FolButton followStatus={followStatus} onClick={changeStatus}>
        {innerText}
      </FolButton>